tailcall-fixtures/fixtures/configs/user-posts.graphql

```diff
@@ -18,6 +18,7 @@ type User {
   email: String!
   phone: String
   website: String @expr(body: "/users/website/{{.value.username}}")
+  posts: [Post] @http(path: "/posts")
```

Comment:
add comments instead.

Reply:
@tusharmath do we still need this PR? Since @meskill is already working on fixing the issue

Reply:
I'll pick this up